python/kserve/pyproject.toml

```diff
@@ -61,6 +66,8 @@ azure-storage-blob = { version = "^12.10.0", optional = true }
 azure-storage-file-share = { version = "^12.7.0", optional = true }
 azure-identity = { version = "^1.8.0", optional = true }
 boto3 = { version = "^1.21.0", optional = true }
+transformers = { version = "^4.37.0", optional = true }
+torch = { version = "^2.3.0+cpu", optional = true, source = "pytorch"}
```

Comment:
I think the hf storage can be standalone and does not need to depend on kserve package